src/Meshes/mesh.vertexData.ts

```diff
@@ -504,14 +504,15 @@ export class VertexData {
      * @returns the modified VertexData
      */
     public merge(others: VertexData | VertexData[], use32BitsIndices = false) {
-        return runCoroutineSync(this._mergeCoroutine(others, use32BitsIndices, false));
+        const vertexDatas: [vertexData: VertexData, transform?: Matrix][] = Array.isArray(others) ? others.map((other) => [other, undefined]) : [[others, undefined]];

```

Comment:
Why not keeping the complexity in _mergeCoroutine ? as it would be share with the async version ???

Reply:
All this is doing is adding undefined transform matrices to the args. I have to do this to not create a breaking change in this public API.

Reply:
My bad I was thinking of the Mesh static function did not even notice it was in vertex data ... :-)